src/app/screens/connectors/ConnectAlby/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import Button from "~/app/components/Button";
+import api from "~/common/lib/api";
+import msg from "~/common/lib/msg";
+
+export default function ConnectAlby() {
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate();
+  const { t: tCommon } = useTranslation("common");
+  const { t } = useTranslation("translation", {
+    keyPrefix: "choose_path.alby",
+  });
+  async function connectAlby() {
+    setLoading(true);
+    const name = "Alby";
+    const initialAccount = {
+      name,
+      config: {},
+      connector: "alby" as const,
+    };
+
+    try {
+      const validation = await api.validateAccount(initialAccount);
+      if (validation.valid) {
+        if (!validation.oAuthToken) {
+          throw new Error("No oAuthToken returned");
+        }
+
+        const account = {
+          ...initialAccount,
+          config: {
+            ...initialAccount.config,
+            oAuthToken: validation.oAuthToken,
+          },
+        };
+
+        const addResult = await msg.request("addAccount", account);
+        if (addResult.accountId) {
+          await msg.request("selectAccount", {
```

Comment:
Use API or useAccounts()?

Reply:
Will add a TODO (should be updated on all connectors)